Automattic/woocommerce-payments
# Fix WooPay phone number on blocks checkout

Fixes 1610-gh-Automattic/woopay

#### Changes proposed in this Pull Request

This PR fixes a bug that happens when signing up to WooPay with blocks checkout when the user filled out a phone number without country code.









#### Testing instructions





* Add a product to cart.
* Go to blocks checkout.
* Type an e-mail address that doesn't exist on WooPay.
* Before clicking on `Save my information for a faster and secure checkout` checkbox, fill out a phone number without the country code.
* Click on the `Save my information for a faster and secure checkout` checkbox.
* Do not change the phone number.
* Place Order.
* The user should be created on WooPay with the `+1` country code.

-------------------


**Post merge**
